examples/language-modeling/run_language_modeling.py

```diff
@@ -143,6 +149,16 @@ def get_dataset(
 ):
     def _dataset(file_path):
         if args.line_by_line:
+            if args.chinese_ref_file:
+                if not args.wwm or args.mlm:
```

Comment:
The test is not consistent with the error message that wants both of those to be True.

Reply:
fix